### PR TITLE
26/10/2018 balance patch

### PR DESCRIPTION
**Pull request by Apofenas**

### CODE REVIEW

File: units/INS3004/INS3004_unit.bp [hunk 1]
```diff
@@ -719,69 +719,69 @@ UnitBlueprint {
...
-        HealthMult = 0.9,
-        MassMult = 0.9,
-        ReclaimTimeMultiplier = 1,
-        WreckageLayers = {
-            Air = false,
-            Land = false,
-            Seabed = true,
-            Sub = true,
-            Water = true,
+-- T3 heavy destroyer
```
Comment (CookieNoob) [inline units/INS3004/INS3004_unit.bp]: why did you add the script of the unit to the blueprint??

**Comment by CookieNoob** [comment]
I merged this PR into a test branch (so I can do changes as well) and made a PR to merge it into develop: https://github.com/FAForever/nomads/pull/334

**Comment by Apofenas** [comment]
i was chatting with you exactly when i was copying naval stuff

File: units/INS3004/INS3004_unit.bp (hunk 1 above)
Comment (Apofenas) [reply units/INS3004/INS3004_unit.bp]: `-- T3 Heavy destroyer

UnitBlueprint {
    AI = {
        GuardReturnRadius = 130,
        TargetBones = {
            'INS3004',
            'CannonTurret1',
            'CannonTurret2',
            'RocketLauncher',
            'GunRight',
            'GunLeft',
        },
    },
    Audio = {
        AmbientMove = Sound {
            Bank = 'UES',
            Cue = 'UES0302_Move_Loop',
            LodCutoff = 'UnitMove_LodCutoff',
        },
        Killed = Sound {
            Bank = 'UESDestroy',
            Cue = 'UES0302_Destroy',
            LodCutoff = 'UnitMove_LodCutoff',
        },
        StartMove = Sound {
            Bank = 'UES',
            Cue = 'UES0302_Move_Start',
            LodCutoff = 'UnitMove_LodCutoff',
        },
        StopMove = Sound {
            Bank = 'UES',
            Cue = 'UES0302_Move_Stop',
            LodCutoff = 'UnitMove_LodCutoff',
        },
        UISelection = Sound {
            Bank = 'NomadsInterface',
            Cue = 'Nomads_Select_Naval',
            LodCutoff = 'UnitMove_LodCutoff',
        },
    },
    Buffs = {
        Regen = {
            Level1 = 5,
            Level2 = 10,
            Level3 = 15,
            Level4 = 20,
            Level5 = 25,
        },
    },
    BuildIconSortPriority = 20,
    Categories = {
        'SELECTABLE',
        'BUILTBYTIER3FACTORY',
        'NOMADS',
        'MOBILE',
        'NAVAL',
        'TECH3',
        'DIRECTFIRE',
        'ANTIAIR',
        'VISIBLETORECON',
        'RECLAIMABLE',
        'SHOWATTACKRETICLE',
        'DESTROYER',
        'INTELLIGENCE',
        'RADAR',
        'SONAR',
        'OVERLAYANTIAIR',
        'OVERLAYRADAR',
        'OVERLAYSONAR',
        'OVERLAYDIRECTFIRE',
    },
    CollisionOffsetY = -0.375,
    CollisionOffsetZ = 0.8,
    Defense = {
        AirThreatLevel = 16,
        ArmorType = 'Normal',
        EconomyThreatLevel = 0,
        Health = 34000,
        MaxHealth = 34000,
        RegenRate = 0,
        SubThreatLevel = 45,
        SurfaceThreatLevel = 180,
    },
    Description = '<LOC ins3004_desc>Heavy Destroyer',
    Display = {
        Abilities = {
            '<LOC ability_aa>Anti-Air',
            '<LOC ability_underwater_railgun>Submerged Railgun',
            '<LOC ability_radar>Radar',
            '<LOC ability_sonar>Sonar',
        },
--[[        AnimationDeath = {
            {
                Animation = '/units/INS3004/INS3004_ADeath01.sca',
                Rate = 0.03,
                Weight = 100,
            },
            {
                Animation = '/units/INS3004/INS3004_ADeath01.sca',
                Rate = 0.04,
                Weight = 100,
            },
            {
                Animation = '/units/INS3004/INS3004_ADeath02.sca',
                Rate = 0.03,
                Weight = 100,
            },
            {
                Animation = '/units/INS3004/INS3004_ADeath02.sca',
                Rate = 0.04,
                Weight = 100,
            },
            {
                Animation = '/units/INS3004/INS3004_ADeath02.sca',
                Rate = 0.05,
                Weight = 100,
            },
            {
                Animation = '/units/INS3004/INS3004_ADeath03.sca',
                Rate = 0.04,
                Weight = 100,
            },
            {
                Animation = '/units/INS3004/INS3004_ADeath03.sca',
                Rate = 0.05,
                Weight = 100,
            },
            {
                Animation = '/units/INS3004/INS3004_ADeath04.sca',
                Rate = 0.04,
                Weight = 100,
            },
            {
                Animation = '/units/INS3004/INS3004_ADeath04.sca',
                Rate = 0.05,
                Weight = 100,
            },
        },]]
        IdleEffects = {
            Water = {
                Effects = {
                    {
                        Bones = {
                            'INS3004',
                        },
                        Scale = 4,
                        Type = 'SeaIdle01',
                    },
                },
            },
        },
        Mesh = {
            IconFadeInZoom = 130,
            LODs = {
                {
                    AlbedoName = '/units/INS3004/INS3004_Albedo.dds',
                    LODCutoff = 600,
                    MeshName = '/units/INS3004/INS3004_LOD0.scm',
                    NormalsName = '/units/INS3004/INS3004_normalsTS.dds',
                    Scrolling = true,
                    ShaderName = 'NomadsUnit',
                    SpecularName = '/units/INS3004/INS3004_SpecTeam.dds',
                },
            },
        },
        MovementEffects = {
            Water = {
                Effects = {
                    {
                        Bones = {
                            'WakeLeft',
                        },
                        Scale = 2.5,
                        Type = 'LeftFrontWake',
                    },
                    {
                        Bones = {
                            'WakeRight',
                        },
                        Scale = 2.5,
                        Type = 'RightFrontWake',
                    },{
                        Bones = {
                            'wake',
                        },
                        Scale = 1.5,
                        Type = 'BackWake',
                    },
                    {
                        Bones = {
                            'WakeBack',
                        },
                        Scale = 1.5,
                        Type = 'BackWake',
                    },
                },
            },
        },
        PlaceholderMeshName = 'UXS0005',
        SpawnRandomRotation = false,
        UniformScale = 0.65,
    },
    Economy = {
        BuildCostEnergy = 63000,
        BuildCostMass = 8000,
        BuildTime = 25200,
    },
    Footprint = {
        SizeX = 4,
        SizeZ = 12,
    },
    General = {
        Category = 'Ship',
        Classification = 'RULEUC_MilitaryShip',
        CommandCaps = {
            RULEUCC_Attack = true,
            RULEUCC_CallTransport = false,
            RULEUCC_Capture = false,
            RULEUCC_Guard = true,
            RULEUCC_Move = true,
            RULEUCC_Nuke = false,
            RULEUCC_Patrol = true,
            RULEUCC_Reclaim = false,
            RULEUCC_Repair = false,
            RULEUCC_RetaliateToggle = true,
            RULEUCC_Stop = true,
            RULEUCC_Transport = false,
        },
        FactionName = 'Nomads',
        Icon = 'sea',
        OrderOverrides = {
        },
        TechLevel = 'RULEUTL_Secret',
        ToggleCaps = {
        },
        UnitName = '<LOC ins3004_name>Juggernaut',
        UnitWeight = 1,
    },
    Intel = {
        RadarRadius = 50,
        SonarRadius = 30,
        VisionRadius = 50,
        WaterVisionRadius = 50,
    },
    Interface = {
        HelpText = '<LOC ins3004_help>This ship is a little bit thin in an armor, but carries great fire power of 2 heavy long range plasma turrets, two smaller stingray cannons, flak AA defense and a submerged railgun.',
    },
    LifeBarHeight = 0.075,
    LifeBarOffset = 6.5,
    LifeBarSize = 4.5,
    Physics = {
        BackUpDistance = 10,
        BankingSlope = 0,
        BuildOnLayerCaps = {
            LAYER_Air = false,
            LAYER_Land = false,
            LAYER_Orbit = false,
            LAYER_Seabed = false,
            LAYER_Sub = false,
            LAYER_Water = true,
        },
        CatchUpAcc = 10,
        DragCoefficient = 0.2,
        Footprint = {
            SizeX = 2.3,
            SizeY = 2.35,
            SizeZ = 13,
        },
        MaxAcceleration = 4,
        MaxBrake = 4,
        MaxSpeed = 4,
        MaxSpeedReverse = 4,
        MaxSteerForce = 5,
        MeshExtentsX = 2.25,
        MeshExtentsY = 2,
        MeshExtentsZ = 12,
        MinSpeedPercent = 0,
        MotionType = 'RULEUMT_Water',
        TurnRadius = 25,
        TurnRate = 35,
    },
    SelectionSizeX = 1.6,
    SelectionSizeZ = 8.2,
    SelectionThickness = 0.2,
    SizeX = 2.8,
    SizeY = 2.1,
    SizeZ = 12,
    StrategicIconName = 'icon_ship3_directfire',
    StrategicIconSortPriority = 145,
    Veteran = {
        Level1 = 30,
        Level2 = 60,
        Level3 = 90,
        Level4 = 120,
        Level5 = 150,
    },
    Weapon = {
        {
            AboveWaterTargetsOnly = true,
            Audio = {
                Fire = Sound {
                    Bank = 'NomadsWeapons',
                    Cue = 'PlasmaCannon_Muzzle',
                    LodCutoff = 'Weapon_LodCutoff',
                },
            },
            BallisticArc = 'RULEUBA_LowArc',
            CollideFriendly = false,
            Damage = 200,
            DamageRadius = 2,
            DamageType = 'Normal',
            DisplayName = 'Plasma cannon',
            FireTargetLayerCapsTable = {
                Water = 'Land|Water|Seabed',
            },
            FiringRandomness = 0.3,
            FiringTolerance = 2,
            Label = 'MainTurret1',
            MaxRadius = 128,
            MuzzleSalvoDelay = 0,
            MuzzleSalvoSize = 1,
            MuzzleVelocity = 40,
            NotExclusive = true,
            ProjectileId =  '/projectiles/NPlasmaProj1/NPlasmaProj1_proj.bp',
            ProjectilesPerOnFire = 3,
            RackBones = {
                {
                    MuzzleBones = {
                        'CannonTurret1_Muzzle1',
                    },
                    RackBone = 'CannonTurret1_Muzzle1',
                },
                {
                    MuzzleBones = {
                        'CannonTurret1_Muzzle2',
                    },
                    RackBone = 'CannonTurret1_Muzzle2',
                },
                {
                    MuzzleBones = {
                        'CannonTurret1_Muzzle3',
                    },
                    RackBone = 'CannonTurret1_Muzzle3',
                },
            },
            RackFireTogether = true,
            RackRecoilDistance = -0.4,
            RackReloadTimeout = 0,
            RackSalvoChargeTime = 0,
            RackSalvoReloadTime = 0,
            RackSalvoSize = 1,
            RangeCategory = 'UWRC_DirectFire',
            RateOfFire = 0.5,
            SlavedToBody = true,
            SlavedToBodyArcRange = 155,
            TargetCheckInterval = 3,
            TargetPriorities = {
                'SPECIALHIGHPRI',
                'NAVAL MOBILE',
                'SPECIALLOWPRI',
                'ALLUNITS',
            },
            TargetRestrictDisallow = 'UNTARGETABLE',
            TrackingRadius = 1.15,
            TurretBoneMuzzle = 'CannonTurret1_Muzzle1',
            TurretBonePitch = 'CannonTurret1_Pitch',
            TurretBoneYaw = 'CannonTurret1',
            TurretDualManipulators = false,
            TurretPitch = 20,
            TurretPitchRange = 25,
            TurretPitchSpeed = 10,
            TurretYaw = 0,
            TurretYawRange = 150,
            TurretYawSpeed = 70,
            Turreted = true,
            WeaponCategory = 'Direct Fire Naval',
            WeaponRepackTimeout = 0,
            WeaponUnpacks = false,
        },
        {
            AboveWaterTargetsOnly = true,
            Audio = {
                Fire = Sound {
                    Bank = 'NomadsWeapons',
                    Cue = 'PlasmaCannon_Muzzle',
                    LodCutoff = 'Weapon_LodCutoff',
                },
            },
            BallisticArc = 'RULEUBA_LowArc',
            CollideFriendly = false,
            Damage = 200,
            DamageRadius = 2,
            DamageType = 'Normal',
            DisplayName = 'Plasma cannon',
            FireTargetLayerCapsTable = {
                Water = 'Land|Water|Seabed',
            },
            FiringRandomness = 0.3,
            FiringTolerance = 2,
            Label = 'MainTurret2',
            MaxRadius = 128,
            MinRadius = 10,
            MuzzleSalvoDelay = 0,
            MuzzleSalvoSize = 1,
            MuzzleVelocity = 40,
            NotExclusive = true,
            ProjectileId =  '/projectiles/NPlasmaProj1/NPlasmaProj1_proj.bp',
            ProjectilesPerOnFire = 3,
            RackBones = {
                {
                    MuzzleBones = {
                        'CannonTurret2_Muzzle1',
                    },
                    RackBone = 'CannonTurret2_Muzzle1',
                },
                {
                    MuzzleBones = {
                        'CannonTurret2_Muzzle2',
                    },
                    RackBone = 'CannonTurret2_Muzzle2',
                },
                {
                    MuzzleBones = {
                        'CannonTurret2_Muzzle3',
                    },
                    RackBone = 'CannonTurret2_Muzzle3',
                },
            },
            RackFireTogether = true,
            RackRecoilDistance = -0.4,
            RackReloadTimeout = 0,
            RackSalvoChargeTime = 0,
            RackSalvoReloadTime = 0,
            RackSalvoSize = 1,
            RangeCategory = 'UWRC_DirectFire',
            RateOfFire = 0.5,
            SlavedToBody = true,
            SlavedToBodyArcRange = 155,
            TargetCheckInterval = 3,
            TargetPriorities = {
                'SPECIALHIGHPRI',
                'NAVAL MOBILE',
                'SPECIALLOWPRI',
                'ALLUNITS',
            },
            TargetRestrictDisallow = 'UNTARGETABLE',
            TrackingRadius = 1.15,
            TurretBoneMuzzle = 'CannonTurret2_Muzzle1',
            TurretBonePitch = 'CannonTurret2_Pitch',
            TurretBoneYaw = 'CannonTurret2',
            TurretDualManipulators = false,
            TurretPitch = 20,
            TurretPitchRange = 25,
            TurretPitchSpeed = 10,
            TurretYaw = 0,
            TurretYawRange = 150,
            TurretYawSpeed = 70,
            Turreted = true,
            WeaponCategory = 'Direct Fire Naval',
            WeaponRepackTimeout = 0,
            WeaponUnpacks = false,
        },
        {
             AboveWaterTargetsOnly = true,
             Audio = {
                Fire = Sound {
                    Bank = 'NomadsWeapons',
                    Cue = 'Stingray_Muzzle',
                    LodCutoff = 'Weapon_LodCutoff',
                },
            },
            BallisticArc = 'RULEUBA_LowArc',
            CollideFriendly = false,
            Damage = 20,
            DamageRadius = 1,
            DamageType = 'Normal',
            DisplayName = 'Stingray',
            FireTargetLayerCapsTable = {
                Water = 'Land|Water|Seabed',
            },
            FiringRandomness = 0.2,
            FiringTolerance = 2,
            Label = 'SideTurret1',
            MaxRadius = 50,
            MuzzleSalvoDelay = 0,
            MuzzleSalvoSize = 1,
            MuzzleVelocity = 40,
            NotExclusive = true,
            ProjectileId = '/projectiles/NStingray1/NStingray1_proj.bp',
            ProjectileLifetime = 4,
            ProjectileLifetimeUsesMultiplier = 1.15,
            ProjectilesPerOnFire = 1,
            RackBones = {
                {
                    MuzzleBones = {
                        'GunLeft_Muzzle',
                    },
                    RackBone = 'GunLeft_Muzzle',
                },
            },
            RackFireTogether = false,
            RackRecoilDistance = -0.1,
            RackReloadTimeout = 10,
            RackSalvoChargeTime = 0,
            RackSalvoReloadTime = 0,
            RackSalvoSize = 1,
            RackSlavedToTurret = false,
            RangeCategory = 'UWRC_DirectFire',
            RateOfFire = 3.333,
            SlavedToBody = true,
            SlavedToBodyArcRange = 90,
            TargetCheckInterval = 0.5,
            TargetPriorities = {
                'SPECIALHIGHPRI',
                'SPECIALLOWPRI',
                'ALLUNITS',
            },
            TargetRestrictDisallow = 'HIGHALTAIR,UNTARGETABLE',
            TurretBoneMuzzle = 'GunLeft_Muzzle',
            TurretBonePitch = 'GunLeft_Pitch',
            TurretBoneYaw = 'GunLeft',
            TurretDualManipulators = false,
            TurretPitch = 0,
            TurretPitchRange = 45,
            TurretPitchSpeed = 90,
            TurretYaw = 90,
            TurretYawRange = 100,
            TurretYawSpeed = 90,
            Turreted = true,
            WeaponCategory = 'Direct Fire',
        },
        {
             AboveWaterTargetsOnly = true,
             Audio = {
                Fire = Sound {
                    Bank = 'NomadsWeapons',
                    Cue = 'Stingray_Muzzle',
                    LodCutoff = 'Weapon_LodCutoff',
                },
            },
            BallisticArc = 'RULEUBA_LowArc',
            CollideFriendly = false,
            Damage = 20,
            DamageRadius = 1,
            DamageType = 'Normal',
            DisplayName = 'Stingray',
            FireTargetLayerCapsTable = {
                Water = 'Land|Water|Seabed',
            },
            FiringRandomness = 0.2,
            FiringTolerance = 2,
            Label = 'SideTurret2',
            MaxRadius = 50,
            MuzzleSalvoDelay = 0,
            MuzzleSalvoSize = 1,
            MuzzleVelocity = 40,
            NotExclusive = true,
            ProjectileId = '/projectiles/NStingray1/NStingray1_proj.bp',
            ProjectileLifetime = 4,
            ProjectileLifetimeUsesMultiplier = 1.15,
            ProjectilesPerOnFire = 1,
            RackBones = {
                {
                    MuzzleBones = {
                        'GunRight_Muzzle',
                    },
                    RackBone = 'GunRight_Muzzle',
                },
            },
            RackFireTogether = false,
            RackRecoilDistance = -0.1,
            RackReloadTimeout = 10,
            RackSalvoChargeTime = 0,
            RackSalvoReloadTime = 0,
            RackSalvoSize = 1,
            RackSlavedToTurret = false,
            RangeCategory = 'UWRC_DirectFire',
            RateOfFire = 3.333,
            SlavedToBody = true,
            SlavedToBodyArcRange = 270,
            TargetCheckInterval = 0.5,
            TargetPriorities = {
                'SPECIALHIGHPRI',
                'SPECIALLOWPRI',
                'ALLUNITS',
            },
            TargetRestrictDisallow = 'HIGHALTAIR,UNTARGETABLE',
            TurretBoneMuzzle = 'GunRight_Muzzle',
            TurretBonePitch = 'GunRight_Pitch',
            TurretBoneYaw = 'GunRight',
            TurretDualManipulators = false,
            TurretPitch = 0,
            TurretPitchRange = 45,
            TurretPitchSpeed = 90,
            TurretYaw = 270,
            TurretYawRange = 100,
            TurretYawSpeed = 90,
            Turreted = true,
            WeaponCategory = 'Direct Fire',
        },
        {
            AlwaysRecheckTarget = true,
            Audio = {
                Fire = Sound {
                    Bank = 'XSB_Weapon',
                    Cue = 'XSB2205_Cavitation_Torpedo',
                    LodCutoff = 'Weapon_LodCutoff'
                },
            },
            BallisticArc = 'RULEUBA_None',
            CollideFriendly = false,
            Damage = 400,
            DamageType = 'Normal',
            DisplayName = 'Submerged rail gun',
            FireTargetLayerCapsTable = {
                Water = 'Water|Seabed|Sub',
            },
            FiringRandomness = 0,
            FiringTolerance = 0,
            HeadingArcCenter = 0,
            HeadingArcRange = 160,
            Label = 'RailGun',
            LeadTarget = true,
            MaxRadius = 60,
            MinRadius = 3,
            MuzzleSalvoDelay = 0,
            MuzzleSalvoSize = 1,
            MuzzleVelocity = 80,
            NotExclusive = true,
            ProjectileId = '/projectiles/NUnderwaterRailProj2/NUnderwaterRailProj2_proj.bp',
            ProjectileLifetime = 0.7,
            ProjectilesPerOnFire = 1,
            RackBones = {
                {
                    MuzzleBones = {
                        'Railgun_Muzzle',
                    },
                    RackBone = 'Railgun_Pitch',
                },
            },
            RackFireTogether = false,
            RackRecoilDistance = 0,
            RackReloadTimeout = 0,
            RackSalvoChargeTime = 0,
            RackSalvoReloadTime = 0,
            RackSalvoSize = 1,
            RackSlavedToTurret = false,
            RangeCategory = 'UWRC_AntiNavy',
            RateOfFire = 0.4,
            SlavedToBody = true,
            SlavedToBodyArcRange = 360,
            TargetCheckInterval = 1,
            TargetPriorities = {
                'SPECIALHIGHPRI',
                'MOBILE',
                'STRUCTURE DEFENSE',
                'SPECIALLOWPRI',
                'ALLUNITS',
            },
            TargetRestrictDisallow = 'UNTARGETABLE, HOVER',
            TargetCheckInterval = 1,
            TrackingRadius = 1.15,
            TurretBoneMuzzle = 'Railgun_Muzzle',
            TurretBonePitch = 'Railgun_Pitch',
            TurretBoneYaw = 'Railgun',
            TurretDualManipulators = false,
            TurretPitch = -30,
            TurretPitchRange = 40,
            TurretPitchSpeed = 35,
            TurretYaw = 0,
            TurretYawRange = 360,
            TurretYawSpeed = 90,
            Turreted = true,
            WeaponCategory = 'Anti Navy',
            WeaponRepackTimeout = 0,
            WeaponUnpacks = false,
        },
        {
            AlwaysRecheckTarget = true,
            Audio = {
                Fire = Sound {
                    Bank = 'URLWeapon',
                    Cue = 'URL0402_Missile_Nanite',
                    LodCutoff = 'Weapon_LodCutoff',
                },
            },
            BallisticArc = 'RULEUBA_None',
            CannotAttackGround = true,
            CollideFriendly = false,
            Damage = 100,
            DamageFriendly = false,
            DamageRadius = 3,
            DamageType = 'Normal',
            DetonatesAtTargetHeight = true,
            DisplayName = 'Rocket flak',
            FireTargetLayerCapsTable = {
                Water = 'Air',
            },
            FiringRandomness = 1,
            FiringTolerance = 2,
            LeadTarget = true,
            Label = 'AATurret',
            MaxRadius = 30,
            MuzzleSalvoDelay = 0,
            MuzzleSalvoSize = 1,
            MuzzleVelocity = 28,
            NotExclusive = true,
            ProjectileId = '/projectiles/NRocketFlakProj1/NRocketFlakProj1_proj.bp',
            ProjectileLifetime = 3,
            ProjectileLifetimeUsesMultiplier = 1.25,
            ProjectilesPerOnFire = 2,
            RackBones = {
                {
                    MuzzleBones = {
                        'RocketLauncher_Muzzle1',
                    },
                    RackBone = 'RocketLauncher_Revolver1',
                },
                {
                    MuzzleBones = {
                        'RocketLauncher_Muzzle2',
                    },
                    RackBone = 'RocketLauncher_Revolver2',
                },
            },
            RackFireTogether = false,
            RackRecoilDistance = 0,
            RackReloadTimeout = 0,
            RackSalvoChargeTime = 0,
            RackSalvoReloadTime = 0,
            RackSalvoSize = 1,
            RackSlavedToTurret = false,
            RangeCategory = 'UWRC_AntiAir',
            RateOfFire = 2.5,
            TargetCheckInterval = 0.3,
            TargetPriorities = {
                'SPECIALHIGHPRI',
                'EXPERIMENTAL',
                'AIR MOBILE HIGHPRIAIR',
                'AIR MOBILE TECH3 BOMBER',
                'AIR MOBILE BOMBER',
                'AIR MOBILE GROUNDATTACK',
                'AIR MOBILE TRANSPORTATION',
                'AIR MOBILE',
                'SPECIALLOWPRI',
                'ALLUNITS',
            },
            TargetRestrictDisallow = 'UNTARGETABLE',
            TrackingRadius = 1.4,
            TurretBoneMuzzle = 'RocketLauncher_Pitch',
            TurretBonePitch = 'RocketLauncher_Pitch',
            TurretBoneYaw = 'RocketLauncher',
            TurretDualManipulators = false,
            TurretPitch = 40,
            TurretPitchRange = 50,
            TurretPitchSpeed = 180,
            TurretYaw = 0,
            TurretYawRange = 180,
            TurretYawSpeed = 180,
            Turreted = true,
            WeaponCategory = 'Anti Air',
            WeaponRepackTimeout = 0,
            WeaponUnpacks = false,
        },
    },
    Wreckage = {
        Blueprint = '/props/DefaultWreckage/DefaultWreckage_prop.bp',
        EnergyMult = 0,
        HealthMult = 0.9,
        MassMult = 0.9,
        ReclaimTimeMultiplier = 1,
        WreckageLayers = {
            Air = false,
            Land = false,
            Seabed = true,
            Sub = true,
            Water = true,
        },
    },
}`
